schematic_api/api/routes.py

```diff
@@ -182,7 +103,7 @@ def convert_df_to_csv(self, df, file_name):
         """
 
         # convert dataframe to a temporary csv file
-        temp_dir = tempfile.gettempdir()
+        temp_dir = create_temp_folder(path=tempfile.gettempdir())
```

Comment:
This was failing with high concurrency as `tempfile.gettempdir()` only returned back `/tmp`, and for tests where the name of the uploaded file was the same was causing errors. By doing this we are guaranteed to get back a unique temporary directory.

Reply:
I wonder if this impacts deployment, not sure if the /tmp directory is mounted in any way (probably not)